prow/repoowners/repoowners_test.go

```diff
@@ -1265,6 +1265,9 @@ reviewers:
 			t.Errorf("result '%s' is differ from expected: '%s'", s, test.expected)
 		}
 		simple, err := LoadSimpleConfig(b)
+		if err != nil {
+			t.Errorf("unexpected error when load simple config")
```

Comment:
Same thing in this file (include the error message)

Reply:
I have done